utils/make-pkgconfig.swift

```diff
@@ -46,24 +46,21 @@ extension String: Error {
 }
 
 func makeFile() throws {
-  let pkgConfigPath = "/usr/local/lib/pkgconfig"
+  guard let brew = which("brew") else { throw "Missing callable `brew` instance." }
```

Comment:
We cannot assume the LLVM installation was done via `brew` because that breaks Linux users. This is why the CI is failing.

Reply:
I see.
I'm going to add platform branching. (`#available`)

For the Linux, I don't know well about it. Just keep `/usr/local/` for Linux with hoping some Linux expert to come up to fix it.

For the macOS, I think `/usr/local` can cover only x86 Homebrew. (maybe some other package managers?) How about calling `brew --prefix` first and falling back to `/usr/local`?

Reply:
I'm not sure how useful `#available` will be since [brew](https://brew.sh) is now available for Linux I think, so in theory someone could install it that way.

As for trying brew first then falling back to `/usr/local`, that's probably fine. @segiddins @dduan what do you think?

Reply:
I agree to that `#available` is not necessary.